Code/Source/svFSI/consts.h

```diff
@@ -460,6 +460,13 @@ std::ostream& operator<<(typename std::enable_if<std::is_enum<T>::value, std::os
     return stream << static_cast<typename std::underlying_type<T>::type>(e);
 }
 
+//// Mechanical configurations
+enum class MechanicalConfigurationType
+{
+  t_ref, // reference configuration
```

Comment:
A good enum class name but what does `t_` mean? Why not just `reference`, `previous` and `next` ?

Reply:
I was going to do `ref`, `new`, and `old`, but `new` wasn't allowed because it's the C++ keyword for dynamic memory allocation. I figured `t_` just means time or timestep, but I admit it's not the best. I want to do `new` and `old` to be analogous to the state variable names in svFSIPlus (i.e. `Ao`/`An`, `Yo`/`Yn`, `Do`/`Dn`). Open to other ideas!

Reply:
Don't hesitate to use long descriptive variable names. Names like `An` have no meaning to a new developer. 

`new_time_step`, `old_time_step` and 'ref_time_step` are good if that is what the enums represent.

Reply:
Okay great, let's go with `reference`, `new_timestep`, and `old_timestep`. I think these are the most natural when one is talking about mechanical configuration (e.g. "integrate in the reference configuration"/"get the normal vector in the new timestep configuration")

Reply:
Good!